### DISCUSSION TITLE
How to add new options to the plugin config?

### DISCUSSION THREAD

**Discussion by prsh11** [NvChad/NvChad#1732] [Q&A] [answered]
Adding the following to cmp config
```lua
completion = {
    completeopt = 'menu,menuone,noinsert'
  }
  ```
This option does not exist in the current config. So I think `override_options` parameter is not working. 
```lua
["hrsh7th/nvim-cmp"] = {
      override_options = {
        completion = {
          completeopt = "menu,menuone,noinsert"
          }
        }
}
```
When I add it directly to the core config, it works flawlessly but does not work when used in the chadrc.lua

**Comment by prsh11** [comment]


**Reply by siduck** [reply]
thats not the right way @prsh11

**Comment by siduck** [comment, marked as answer]
https://user-images.githubusercontent.com/59060246/210215722-187508b8-542f-465b-b3af-9f264296cb31.mp4

@prsh11 it works with your override code, watch the video

**Reply by prsh11** [reply]
got it. I had the user parameter. I think that's the reason.
